style.css

```diff
@@ -29,3 +30,44 @@ a:hover {
   border-bottom: 0;
 }
 
+h1.title {
+  font-size: 3.1em;
+  line-height: 1.4em;
+}
+
+header#title-block-header {
+  padding: 12em 2em;
+  border-bottom: 2px dotted #aaaaaa;
+}
+
+nav {
+  padding: 6em 0;
+  border-bottom: 2px dotted #aaaaaa;
+
+}
+
+nav::before {
+  content: 'Table of Contents'
+}
```

Comment:
Could we try https://github.com/jgm/pandoc/wiki/How-to-add-a-%22Table-of-Contents%22-title-in-the-HTML-template? 😆

Reply:
Ahh this i like, ok 👍